convert-sg-db.py

```diff
@@ -0,0 +1,131 @@
+
+<p>Note that per the <a href="/workstream-policy">Workstream Policy</a>, the official <a href="/workstream-policy#editor">Editors</a> listed here may have delegated responsibilities and editing duties to deputy editors for any given <a href="/workstream-policy#living-standard">Living Standard</a>. As such, the <a href="/workstream-policy#editor">Editor</a> position listed here is more about "who is the official liaison with the <a href="/sg-agreement#steering-group">Steering Group</a>" and less about "who is editing the document". To get an accounting of who is primarily responsible for day-to-day specification work, you'll be better served by checking the Acknowledgments section of the <a href="/workstream-policy#living-standard">Living Standard</a> in question.</p>"""
+    content += "\n\n<dl>"
+    for workstream in db["workstreams"]:
+        content += "\n <dt>{}</dt>".format(workstream["name"])
+        content += "\n <dd>"
+        content += "\n  <p><strong>Scope:</strong> {}</p>".format(commonmark.commonmark(workstream["scope"])[3:-5]) # Strip leading <p> and trailing </p>\n
+        if len(workstream["editors"]) == 1:
+            content += """\n  <p><strong>Editor:</strong> {0}, <a href="mailto:{1}">{1}</a></p>""".format(workstream["editors"][0]["name"], workstream["editors"][0]["email"])
+        else:
+            content += "\n  <p><strong>Editors:</strong></p>"
+            content += "\n  <ol>"
+            for editor in workstream["editors"]:
+                content += """\n   <li>{0}, <a href="mailto:{1}">{1}</a></p>""".format(editor["name"], editor["email"])
+            content += "\n  </ol>"
+        if (len(workstream["standards"]) == 1):
+            content += """\n  <p><strong>Standard:</strong> <a href="{0}">{1}</a></p>""".format(workstream["standards"][0]["href"], workstream["standards"][0]["name"])
+        else:
+            content += "\n  <p><strong>Standards:</strong></p>"
+            content += "\n  <ol>"
+            for standard in workstream["standards"]:
+                content += """\n   <li><a href="{0}">{1}</a></p>""".format(standard["href"], standard["name"])
+            content += "\n  </ol>"
+        content += "\n </dd>\n"
+    content += "</dl>\n\n"
+    content += copyright
+
+    output = template.replace("@TITLE_GOES_HERE@", "Workstreams")
+    output = output.replace("@CONTENT_GOES_HERE@", content)
+
+    open("whatwg.org/workstreams", "w").write(output)
+
+
+def main():
+    template = open("site-template.html", "r", encoding="utf-8").read()
+    copyright = """<footer>
+ <p><small>Copyright © 2018 WHATWG (Apple, Google, Mozilla, Microsoft). This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.</small></p>
+</footer>"""
```

Comment:
Can we put the copyright in the template?

Reply:
Then we'd have to strip it in convert-policy.py.

Reply:
I suppose we could run a replace on _template_ locally though instead of passing it everywhere.

Reply:
Hmm yeah.

Also 2018 seems wrong. We can probably just use current year for these?

Reply:
I suppose we could, at least until that issue about what year to use is resolved.